ElectronClient/gui/NoteEditor/utils/types.ts

```diff
@@ -42,6 +42,7 @@ export interface NoteBodyEditorProps {
 	disabled: boolean;
 	dispatch: Function;
 	noteToolbar: any;
+	setLocalSearchResultCount: any,
```

Comment:
Shouldn't it be `setLocalSearchResultCount: Function`? Actually if you could set the function signature that would be better like `setLocalSearchResultCount(matches:something):something` (I'm not sure what's the type of `matches` or what the function return. Just return `void` if it returns nothing).

Reply:
Nice one, indeed it should be `setLocalSearchResultCount(count: number): void,`